src/Symfony/Component/Notifier/Bridge/Slack/Block/SlackHeaderBlock.php

```diff
@@ -25,7 +25,7 @@ final class SlackHeaderBlock extends AbstractSlackBlock
 
     public function __construct(string $text)
     {
-        if (\strlen($text) > self::TEXT_LIMIT) {
+        if (mb_strlen($text, 'UTF-8') > self::TEXT_LIMIT) {
```

Comment:
I thought slack count in byte => https://github.com/symfony/symfony/pull/39410/files#r539845790